src/components/wallet/views/balanceDisplay.tsx

```diff
@@ -9,21 +9,26 @@ interface Balance {
   asset_issuer: string
 }
 
-export default function BalanceDisplay() {
+export default function () {
   const balanceRow = (balance: Balance) => {
-    const name = balance.asset_type === 'native' ? 'XLM' : balance.asset_code
+    const assetCode =
+      balance.asset_type === 'native' ? 'XLM' : balance.asset_code
+
     return (
       <div class="balance-row">
-        <div class="asset-code">{name}</div>
-        <div class="balance">{parseFloat(balance.balance).toFixed(2)}</div>
+        <div class="asset-code">{assetCode}</div>
+        <div class="balance">{balance.balance}</div>
```

Comment:
Feels pretty important to include all decimals. We could try and be smarter with the info from TOML files but for now it feels right to include whatever is returned.

Reply:
agreed i'll change

Reply:
oh you did change, 👍